MC/config/PWGHF/ini/GeneratorHF_D2H_ccbar_gap3.ini

```diff
@@ -0,0 +1,7 @@
+### The external generator derives from GeneratorPythia8.
+[GeneratorExternal]
+fileName=${O2DPG_ROOT}/MC/config/PWGHF/external/generator/generator_pythia8_gaptriggered_hf.C
+funcName=GeneratorPythia8GapTriggeredCharm(3, -1.5, 1.5)
+
+[GeneratorPythia8]
+config=${O2DPG_ROOT}/MC/config/PWGHF/pythia8/generator/pythia8_charmhadronic_with_decays.cfg
```

Comment:
At the moment, this configuration is the same as the one for `GeneratorHF_D2H_ccbar.ini`. Moreover, it is not in the list of configurations we plan to test (we said ccbar gap 5, right?).